mmv1/products/redis/Cluster.yaml

```diff
@@ -123,6 +136,25 @@ properties:
     default_from_api: true
     immutable: true
     required: false
+  - !ruby/object:Api::Type::NestedObject
+    name: zoneDistributionConfig
+    description: Zone distribution config for Memorystore Redis cluster.
```

Comment:
The documentation won't automatically provide information to users about a field being immutable. You could put `Immutable.` at the start of the field, and subfield, description(s) to make this clearer to users when they read the docs. [Here's an example](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/cloudbuild_bitbucket_server_config#host_uri) of this being done for an immutable field in other resource.

Reply:
ACK